### PR TITLE
Back to 100vh

### PR DESCRIPTION
**Pull request by Gnito**
Browser bars are a problem with flexbox layout. They might hide save button. This PR changes sideWinder height so that it will initially take 100vh and then (when JS kicks in) window.innerHeight.

### CODE REVIEW

**Comment by Gnito** [comment]
I'm not sure if this is good enough, but if not - we need to figure out some reliable way to detect those browser bars.

**Comment by kpuputti** [comment]
@Gnito I don't think there's a reliable way to detect the browser bars. However, `window.innerWidth` should be correct and Mobile Safari sends a `resize` event to `window` when the bars become visible. So we could dynamically adjust the height with inline styles.

This is exactly what I had done earlier, but the problem was passing the new height in the component children so that the child components would be rendered again.

File: client/app/components/sections/ManageAvailability/ManageAvailability.js [hunk 1]
```diff
@@ -90,7 +94,7 @@ class ManageAvailability extends Component {
     // asynchronously allows the calendar to calculate the height
     // properly.
     // See: https://github.com/airbnb/react-dates/issues/46
-    window.setTimeout(() => {
+    this.calendarTimeout = window.setTimeout(() => {
```
Comment (kpuputti) [inline client/app/components/sections/ManageAvailability/ManageAvailability.js]: Good catch to clear this timeout!

File: client/app/components/sections/ManageAvailability/ManageAvailability.js [hunk 2]
```diff
@@ -99,6 +103,9 @@ class ManageAvailability extends Component {
     }
 
     loadInitialDataIfNeeded(this.props);
+
+    this.setState({ viewportHeight: window.innerHeight }); // eslint-disable-line react/no-set-state
+    window.addEventListener('resize', this.resizeHandler, false);
```
Comment (kpuputti) [inline client/app/components/sections/ManageAvailability/ManageAvailability.js]: No need to add the capture flag, it defaults to `false`.

**Pull request merged**